felix/fv/mtu_test.go

```diff
@@ -39,9 +39,86 @@ var _ = infrastructure.DatastoreDescribe("VXLAN topology before adding host IPs
 		client client.Interface
 	)
 
+	expectMTU := func(mtu int) {
+		for _, felix := range tc.Felixes {
+			EventuallyWithOffset(1, func() string {
+				out, _ := felix.ExecOutput("cat", "/var/lib/calico/mtu")
+				return out
+			}, "60s", "500ms").Should(ContainSubstring(fmt.Sprintf("%d", mtu)))
```

Comment:
Does `ContainSubstring` succeed if output is `13508` but the correct value of mtu should be `1350`?